kruize-clowdapp.yaml

```
@@ -28,6 +28,26 @@ objects:
           limits:
             cpu: ${KRUIZE_CPU_LIMIT}
             memory: ${KRUIZE_MEMORY_LIMIT}
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /health
+            port: 10000
```

```suggestion
            port:  ${{AUTOTUNE_SERVER_PORT}}
```

@saltgen , to confirm could you please make this syntax change?

---

@kgaikwad It looks like the value should not be more than 15 chars, from the error log below,
https://github.com/RedHatInsights/ros-ocp-backend/pull/135#discussion_r1366598939